tests/spec/fs.readdir.spec.js

```diff
@@ -55,3 +55,25 @@ describe('fs.readdir', function() {
     });
   });
 });
+
+/**
+ * fsPromises tests
+ */
+
+describe('fsPromises.readdir', function() {
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+
+  it('should return an error if the path is a file', function(done) {
```

Comment:
To make this more promise-friendly, please remove the `done` from the callback function argument